packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/styles.ts

```diff
@@ -15,7 +15,7 @@ export const sidebar = (width: number, sidebarOffset: number) =>
 export const search = {
   position: "relative",
   marginTop: "large",
```

Comment:
The next 4 changes are to have Search working on the new darker backround. Effectively flipping it to a light-mode search input.